utils/trytes_conversion.go

```diff
@@ -85,6 +87,21 @@ func TrytesToBytes(t giota.Trytes) []byte {
 	return output
 }
 
+func ChunkMessageToTrytesWithStopper(binString string) giota.Trytes {
+	trytes := RunesToTrytes([]rune(binString))
+	return giota.Trytes(trytes + StopperTryte)
+}
+
+func RunesToTrytes(r []rune) giota.Trytes {
```

Comment:
I know you had some issues when trying to preserve the data when doing straight bytes -> trytes without the runes, so perhaps you already took care of this somewhere else in the code, but runes seem to be an alias for the `int32` type to allow programmers to unambiguously know which codepoint (as referenced by utf-8, such as U+0031) you're referring to, and it seems like 1 rune -> 1 tryte (? correct me if I'm reading this wrong). Each tryte has only 3 bits of info, and each rune has 32 bits - so it seems really wasteful to do this unless it's a temporary change than only exists in memory. ie: webinterface sends bytes -> broker stores bytes -> bytes are transformed first to runes and then to trytes -> trytes are uploaded to the tangle.

Perhaps I'm making too many assumptions here though

Reply:
There will be another PR on this same topic in a day or so, to actually store the bytes in the DB and then just convert them to trytes on the fly when we need them.  At the moment, I'm still storing the actual trytes in the DB.  However it's not taking up anymore space than our current implementation, since we're already storing the trytes.  

I want to handle storing the bytes and then converting to trytes on the fly in a separate PR, since there's a lot of places where we use the Message field, and I want to test it pretty thoroughly.

Reply:
To clarify this PR is doing this:
webinterface sends bytes -> broker converts to trytes and stores as trytes -> trytes are uploaded to the tangle.

The next PR will be doing this:
webinterface sends bytes -> broker stores bytes -> bytes are transformed first to runes and then to trytes -> trytes are uploaded to the tangle.